### ISSUE TITLE
Proxy not working

### ISSUE THREAD

**Issue by vidyli** [mock-server/mockserver#994]
# Prerequisite

```
docker run --rm --name mockserver \
-e "MOCKSERVER_ATTEMPT_TO_PROXY_IF_NO_MATCHING_EXPECTATION=true" \
-p 1080:1080  \
mockserver/mockserver  \
-logLevel INFO   \
-serverPort 1080 \
-proxyRemotePort 80 \
-proxyRemoteHost pwa-uat-api.xxx.net
```
My website  use https,  `https://pwa-uat-api.xxx.net`

# Test
```
curl 'http://localhost:1080/graphql?query=query+getAvailableCurrency%7Bcurrency%7Bavailable_currency_codes+code%3Adefault_display_currency_code+symbol%3Adefault_display_currency_symbol+__typename%7D%7D&operationName=getAvailableCurrency&variables=%7B%7D' \
  -H 'authority: pwa-uat-api.xxx.net' \
  -H 'accept: */*' \
  -H 'store: en_us' \
  -H 'user-agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 11_1_0) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/88.0.4324.192 Safari/537.36' \
  -H 'authorization: Bearer undefined' \
  -H 'content-currency: USD' \
  -H 'content-type: application/json' \
  -H 'origin: https://pwa-uat.xxx.net' \
  -H 'sec-fetch-site: same-site' \
  -H 'sec-fetch-mode: cors' \
  -H 'sec-fetch-dest: empty' \
  -H 'referer: https://pwa-uat.xxx.net/' \
  -H 'accept-language: en-US,en;q=0.9,zh-CN;q=0.8,zh;q=0.7' \
  --compressed
```

got 
```
default backend - 404%
```
this request is supposed to return 
```
{
    "data": {
        "currency": {
            "available_currency_codes": [
                "USD"
            ],
            "code": "USD",
            "symbol": "$",
            "__typename": "Currency"
        }
    }
}
```

What I did wrong, please? 

although the document has lots of info, it doesn't follow human instincts, please treat us as stupid as possible,  make it work first, then we will dig if we really need.

**Comment by TinaTiel** [comment]
I'm also wondering this -- my case is slightly different (Spring RestTemplate), but I can see in my application's logs that it is connecting to the proxy on localhost:1080 and making requests through it...But when I pull up the GUI on localhost:1080/mockserver/dashboard I see nothing. 

There has to be something I'm missing in the docs too...Is there some kind of setting that needs to be enabled? All I want to do in my case is record the traffic between my app and the remote server so that I can create the expectations for my tests.

**Comment by jamesdbloom** [comment]
@vidyli @TinaTiel can you please provide the full MockServer logs for both your cases ideally in DEBUG or TRACE level.  Without this it is not possible to see what the issue is.

@TinaTiel the reason your not seeing anything in the dashboard may have been a NullPointException in same case preventing things been shown in the dashboard which has been fixed, so please try the latest version 5.13.0.  If there are any issues like this with the dashboard full logs (ideally TRACE level) would show this.

**Comment by jamesdbloom** [comment]
As this issue is quite old I'm going to close this issue.  If you can provide detailed log providing more detail then I'll happily re-open this issue.
